modules/con_notifications-configuration.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="notifications-configuration-custom-resource-properties_{context}"]
+= NotificationsConfiguration custom resource properties
+
+The `NotificationsConfiguration` resource is a Kubernetes custom resource (CR) that manages notifications in a Kubernetes cluster. In {gitops-title}, you can add templates, triggers, services, and subscription resources to an Argo CD `Notifications` config map by using  the `NotificationsConfiguration` CR.
+
+When you create a cluster in {gitops-title} with notifications enabled, a `NotificationsConfiguration` CR is created by default with the name `default-notifications-configuration`.
+
+Any change made in the existing configuration of the `NotificationsConfiguration` CR is replicated in the Argo CD `Notifications` config map. For example, if the user adds  trigger configuration in the `NotificationsConfiguration` resource, this configuration is read, processed, and updated in the Argo CD `Notifications` config map.
+
+[IMPORTANT]
+====
+Any configuration changes must be updated in  the `default-notifications-configuration` CR. Custom resources created by the users for `NotificationsConfiguration` resource are not supported.
+
+Any modification to the Argo CD `argocd-notifications-cm` config map is overridden by the changes made in the `NotificationsConfiguration` CR.
+====
+
+.`NotificationsConfiguration` custom resource properties
+[cols="1,1,1",options="header"]
+|====
+|**Properties** |**Default** |**Description**
+
+|Templates
+|__<empty>__
+|Templates are used to generate the notification template message.
+
+|Triggers
+|__<empty>__
+|Triggers are used to define the condition when a notification is sent to the user and the list of templates required to generate the message.
+
+|Services
```

Comment:
@Dhruv-Soni11 How do these properties appear int he spec? Please refer to the argo-cd-cr-component-properties.adoc file.

Ask the SME to share the spec and present these properties exactly that way.

Reply:
Fixed as per earlier [comment](https://github.com/openshift/openshift-docs/pull/72540#discussion_r1526270336).